apps/combine-api/Dockerfile

```diff
@@ -1,7 +1,7 @@
 #############
 ### base ###
 #############
-FROM python:3.9-buster as base
+FROM python:3.10-buster as base
```

Comment:
hope this update didn't cause you trouble.

Reply:
I had 3.11 working for a while, but one dependency complained.